messages.js

```diff
@@ -26,19 +26,19 @@ exports.init = function(sbot) {
                      previous ? previous.value.sequence+1 : 1, +new Date())
   }
 
-  function getBase64Nonce() {
-    return crypto.randomBytes(32).toString('base64')
+  function getNonce() {
+    return crypto.randomBytes(32)
```

Comment:
Isn't it a problem to *encode* Buffers into BFE if we can't *decode* them out? https://github.com/ssb-ngi-pointer/ssb-bfe/blob/158c75b827804f2316019e476f03c033130c18f2/index.js#L223

Reply:
Ahh, this is running against an ooold version of db2. Hmm